spec/dummy/log/test.log

```diff
@@ -4,3 +4,110 @@
 Rails::Amqp::Test: test_truth
 -----------------------------
   [1m[35m (0.1ms)[0m  [1m[31mrollback transaction[0m
+  [1m[35m (3.1ms)[0m  [1m[34mSELECT sqlite_version(*)[0m
```

Comment:
You need to ignore log files in gitignore...

Reply:
tks